File: node.js/listUsers.js
var express = require("express");
var app = express();
var fs = require("fs");

var mysql = require("mysql");

app.get('/listUsers', function(req, res) {
    var con = mysql.createConnection({
	host : "localhost",
	user : "root",
	password : "",
	database : "test"
    });
    con.connect(function(err) {
	if (err) {
	    console.log('Error connecting to Db');
	    return;
	}
    });
    
    con.query("SELECT maketxt FROM testtable WHERE idtesttable  = "+req.query.user, function(err, row) {
	res.send(row);
    });
    
    con.end(function(err) {
    });
    
})

var server = app.listen(8081, function() {

    var host = server.address().address
    var port = server.address().port

    console.log("Example app listening at http://%s:%s", host, port)

})

File: golang/src/listUsers.go

package main

import (
    //"database/sql"
    _ "github.com/go-sql-driver/mysql"
	"net/http"
)

func main() {
	
	http.HandleFunc("/apiPing", apiPing)
    http.ListenAndServe(":8082", nil)

   
}

func apiPing(w http.ResponseWriter, r * http.Request){
	
	userId := r.URL.Query().Get("user")
	
 	//db, err := sql.Open("mysql", "root@/test")
    //if err != nil {
    //    panic(err.Error())
    //}
    //defer db.Close()

	//query, err := db.Query("SELECT maketxt FROM testtable WHERE idtesttable = 1")
	//if err != nil {
	//	panic(err.Error())
	//}

	//var col1 int
	//query.Next();
	//err = query.Scan(&col1);
	//if err != nil {
	//	panic(err.Error())
	//}
	//defer query.Close()


    w.Write([]byte(userId));
}